ui/load/vsp.go

```diff
@@ -0,0 +1,193 @@
+package load
```

Comment:
The file is no longer valid as all the details here have been migrated to dcrlibwallet. You will need to rebase your dcrlibwallet PR.